Kajabi/sage-lib
# Button custom content additions

## Description

This PR adds a few more features to the recent custom content in buttons work:

- Adds ability to add a class on the custom content container.
- Adds corresponding features in React Buttons

## Testing in `sage-lib`

- See Button page and the first row of buttons where the custom content example on the end now shows a custom class with some flexbox styling.
- See Storybook > Button > Custom Content for another similar example.

## Testing in `kajabi-products`
(LOW) Updates to buttons with custom content. No impact on existing buttons expected.

## Related
#586
